torch/__init__.py

```diff
@@ -367,46 +367,46 @@ def use_deterministic_algorithms(d):
         * :class:`torch.nn.ConvTranspose2d` when called on CUDA tensor
         * :class:`torch.nn.ConvTranspose3d` when called on CUDA tensor
         * :func:`torch.bmm` when called on sparse-dense CUDA tensors
-        * :func:`torch.__getitem__` backward when `self` is a CPU tensor and
-          ``indices`` is a list of tensors
-        * :func:`torch.index_put` with ``accumulate=True`` when called on a CPU
+        * :func:`torch.Tensor.__getitem__` automatic differentiation with CPU tensors
```

Comment:
Note: This link still doesn't work, but I think that's fine because `__getitem__`/`index` does not seem to have an entry in the docs